pkg/sshd/lock.go

```diff
@@ -38,9 +38,9 @@ func wrapInLock(lck RepositoryLock, repoName string, fn func() error) error {
 		case <-doneCh:
 		}
 	}()
+	defer lck.Unlock(repoName)
```

Comment:
which commit of builder are using..because it used to work fine and was broken in https://github.com/deis/builder/pull/354